### PR TITLE
Improve UI of Tree View

### PR DESCRIPTION
**Pull request by hbjORbj**
- Make creating new branch to `true` by default for `caseElement` and `folderElement`
- Set the folder root name to the name of the repo
- Improve UI for Codemod-first breakdown view

## Final UI

https://user-images.githubusercontent.com/32841130/233669372-2747bbbb-f913-408a-b2f3-1e14aa581e37.mov

### CODE REVIEW

**Review by DmytroHryshyn — APPROVED** [review]

File: src/components/webview/DiffWebviewPanel.ts [hunk 1]
```diff
@@ -77,7 +77,10 @@ export class DiffWebviewPanel extends IntuitaWebviewPanel {
 		}
 
 		const job = this.__jobManager.getJob(jobHash);
-		const jobAccepted = this.__jobManager.isJobAccepted(jobHash);
+		// @TODO
```
Comment (grzpab) [inline src/components/webview/DiffWebviewPanel.ts:80]: what about this TODO?

File: src/components/webview/MainWebviewProvider.ts [hunk 2]
```diff
@@ -48,7 +48,8 @@ import { CaseManager } from '../../cases/caseManager';
 import { SourceControlService } from '../sourceControl';
 
 export const ROOT_ELEMENT_HASH: ElementHash = '' as ElementHash;
-const ROOT_FOLDER_KEY = '/';
+const ROOT_FOLDER_KEY =
```
Comment (grzpab) [inline src/components/webview/MainWebviewProvider.ts:51]: this should be evaluated before doing an operation, not when the extension is being initialized

File: src/extension.ts [hunk 3]
```diff
@@ -403,16 +397,14 @@ export async function activate(context: vscode.ExtensionContext) {
...
-
+		vscode.commands.registerCommand('intuita.createIssue', async () => {
 			const initialData = {
 				repositoryPath: repositoryService.getRemoteUrl(),
 				userId: globalStateAccountStorage.getUserAccount(),
 			};
 
-			const { label } = treeItem;
-			const title = typeof label === 'object' ? label.label : label ?? '';
+			// @TODO
```
Comment (grzpab) [inline src/extension.ts:406]: why a TODO?

File: src/extension.ts [hunk 4]
```diff
@@ -490,6 +520,7 @@ export async function activate(context: vscode.ExtensionContext) {
 							stagedJobs.map(({ hash }) => hash as JobHash),
 						);
 
+						// @should rename this to commitJobs?
```
Comment (grzpab) [inline src/extension.ts:523]: maybe not in this PR, but generally let's do such refactorings when the product is more stable

File: src/components/webview/DiffWebviewPanel.ts (hunk 1 above)
Comment (DmytroHryshyn) [reply src/components/webview/DiffWebviewPanel.ts:80]: We still need remove some code that was depending on jobAccepted, i will do this in next PR

**Review by grzpab — APPROVED** [review]

**Pull request merged**